### DISCUSSION TITLE
Help please

### DISCUSSION THREAD

**Discussion by StephSativa** [community/community#136973] [Code Security]
I know it's a long shot but someone I know got on my phone and reset all my permissions and has access to literally everything.  I'm finding audio recordings of myself  and having access denied. It's through android xml I believe.  Every permission has been changed.  What can I do to fix this other than factory reset ?![Screenshot_20240827_125345_File Viewer.jpg](https://github.com/user-attachments/assets/21872a42-9bd7-4ccb-a547-ce71b1241c50)

How do I edit for myself

**Comment by StephSativa** [comment]
Anyone?

**Comment by MTtankkeo** [comment]
i guess you need to try about https://developer.android.com/tools/adb

**Reply by StephSativa** [reply]
I found  a permissions editor now I just need to find a list of normal permissions to remove the other user controlled ones

**Comment by ghostinhershell** [comment]
Hi @StephSativa,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on this particular issue. Maybe try reaching out to Google support? It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
